app/views/ontologies/selector/selector.html.haml

```diff
@@ -0,0 +1,76 @@
+                    - formats = @formats.drop(1)
+                    %div{'data-ontologies-selector-target': 'table'}
+                        = render TabsContainerComponent.new do |c|         
+                            = c.item(title: 'Categories', selected: true)
+                            = c.item_content do
+                                .chips.categories
+                                    - @filters[:categories][0].each do |item|
+                                        = render ChipsComponent.new(label: item.acronym, id: item.acronym, value: item.id, name: 'categories[]', tooltip: item.name)
+
+                            = c.item(title: 'Groups')
+                            = c.item_content do
+                                .chips.groups
+                                    - @filters[:groups][0].each do |item|
+
+                                        = render ChipsComponent.new(label: item.acronym, id: item.acronym, value: item.id, name: 'groups[]', tooltip: item.name)
+
+                            = c.item(title: 'Format')
+                            = c.item_content do
+                                .chips.format
+                                    - formats.each do |item|
+                                        = render ChipsComponent.new(label: item, id: item, name: 'formats[]', value: item)
+
+                            = c.item(title: 'Natural languages')
+                            = c.item_content do
+                                .chips.natural-language
+                                    - @filters[:naturalLanguage][0].each do |item|
+                                        = render ChipsComponent.new(label: item["acronym"], id: item["acronym"], name: 'naturalLanguage[]', value: item['id'])
+                            
+                            = c.item(title: 'Formality levels')
+                            = c.item_content do
+                                .chips
+                                    - @filters[:hasFormalityLevel][0].each do |item|
+                                        = render ChipsComponent.new(label: item["name"],id: item["acronym"], name: 'formalityLevel[]', value: item["id"])
+                            
+                            = c.item(title: 'Ontology types')
+                            = c.item_content do
+                                .chips
+                                    - @filters[:isOfType][0].each do |item|
+                                        = render ChipsComponent.new(label: item["name"],id: item["acronym"], name: 'isOfType[]', value: item["id"])
+                %input.d-none{type: 'submit', 'data-ontologies-selector-target': 'submit'}
+                = render TurboFrameComponent.new(id: 'selector_results_frame', src: '/ontologies_selector/results') do |container|
+                    - container.loader do
+                        .ontologies-selector-loding
```

Comment:
update the application loader directly instead, we can't have different loader on each page

Reply:
you need to change the LoaderComponent file, and will this new loader will be used every where